### ISSUE TITLE
Error building image

### ISSUE THREAD

**Issue by cmyugovic** [KyroFrCode/moonlight-chrome-tizen-docker#12]
Hi, i got the following error building the image. I went to the previous commit and it worked, so the last commit should be the culprit:

=> ERROR [base 19/25] RUN cmake  -DCMAKE_TOOLCHAIN_FILE=/home/moonlight/  0.6s
------
 > [base 19/25] RUN cmake 	-DCMAKE_TOOLCHAIN_FILE=/home/moonlight/emscripten-release-bundle/emsdk/fastcomp/emscripten/cmake/Modules/Platform/Emscripten.cmake 	-G Ninja 	-S moonlight-chrome-tizen 	-B build:
0.552 CMake Error: The source directory "/home/moonlight/moonlight-chrome-tizen" does not appear to contain CMakeLists.txt.
0.552 Specify --help for usage, or press the help button on the CMake GUI.
------
Dockerfile:58
--------------------
  57 |     
  58 | >>> RUN cmake \
  59 | >>> 	-DCMAKE_TOOLCHAIN_FILE=/home/moonlight/emscripten-release-bundle/emsdk/fastcomp/emscripten/cmake/Modules/Platform/Emscripten.cmake \
  60 | >>> 	-G Ninja \
  61 | >>> 	-S moonlight-chrome-tizen \
  62 | >>> 	-B build
  63 |     RUN cmake --build build
--------------------
ERROR: failed to solve: process "/bin/sh -c cmake \t-DCMAKE_TOOLCHAIN_FILE=/home/moonlight/emscripten-release-bundle/emsdk/fastcomp/emscripten/cmake/Modules/Platform/Emscripten.cmake \t-G Ninja \t-S moonlight-chrome-tizen \t-B build" did not complete successfully: exit code: 1

**Comment by spaceapplesoft** [comment]
Hey,

This repo is missing the source code from which to build. Try this one:
https://github.com/KyroFrCode/moonlight-chrome-tizen

**Comment by Reno333** [comment]
I'm having trouble connectin to the TV.  "sdb connect YOUR_TV_IP" i tried in some ways, but it always fails to connect. 
Can i have some suggestions? I also tried by installing Tizen Studio, but also that software fail to connect to the TV (he found it in the network btw)

**Comment by Trunk68** [comment]
Will this bug be fixed ? I have the same error trying to install the newest version : => ERROR [base 19/25] RUN cmake -DCMAKE_TOOLCHAIN_FILE=/home/moonlight/ 0.6s

**Comment by guba91** [comment]
this error is because we are using tizen 5.0, Wasm has been intriduced in 2020, so older model cannot install *wasm package..
there is a non wasm version?

**Comment by Trunk68** [comment]
No, it was working with the der previous version of wasm without any issues. I can install an run it without any problems, except that there is no 120 fps option.
